src/WebJobs.Extensions.OpenAI.CosmosDBSearch/CosmosDBSearchConfigOptions.cs

```diff
@@ -13,4 +13,42 @@ public class CosmosDBSearchConfigOptions
 
     // This integer is the number of clusters that the inverted file (IVF) index uses to group the vector data
     public int NumLists { get; set; } = 1;
+
+    /// <summary>
+    /// Type of vector index to create.
+    ///     Possible options are:
+    ///         - vector-ivf
+    ///         - vector-hnsw
+    /// </summary>
+    public AzureCosmosDBVectorSearchType Kind { get; set; } =
```

Comment:
Can we update the samples for the usage of this?

Also, README file for the rag-cosmos samples - https://github.com/Azure/azure-functions-openai-extension/blob/main/samples/rag-cosmosdb/README.md

Let me know if you face any issues with running the samples, bit tricky to run dotnet-isolated samples while consuming recent changes when they have not been published to nuget yet.

Reply:
Updated in the host files, and the readme file as well.